### PR TITLE
Viz components - Fix CSP issues in export

### PR DESCRIPTION
**Pull request by alexanderPolosatov**

### CODE REVIEW

**Review by pomahtri — DISMISSED** [review]

File: packages/devextreme/js/viz/core/export.js [hunk 1]
```diff
@@ -311,7 +342,27 @@ export let combineMarkups = function(widgets, options = { }) {
...
+    const { totalWidth, totalHeight } = exportItems;
+
+    const rootElement = wrapItemsToElement(totalWidth,
+        totalHeight,
+        backgroundColor,
+        {
+            options,
+            exportItems,
+            compactView
+        });
```
Comment (ksercs) [inline packages/devextreme/js/viz/core/export.js]: fix indents here please
```javascript
    const rootElement = wrapItemsToElement(
        totalWidth,
        totalHeight,
        backgroundColor,
        {
            options,
            exportItems,
            compactView
        }
    );
```

File: packages/devextreme/testing/tests/DevExpress.viz.core/export.tests.js [hunk 2]
```diff
@@ -683,10 +737,23 @@ QUnit.test('exportFromMarkup. backgroundColor from current theme', function(asse
     }
 });
 
+QUnit.test('exportWidgets method should pass to export markup as string', function(assert) {
```
Comment (ksercs) [inline packages/devextreme/testing/tests/DevExpress.viz.core/export.tests.js]: exportWidgets should pass a dom node to export, not a string, no?

File: packages/devextreme/js/viz/core/export.js (hunk 1 above)
Comment (alexanderPolosatov) [reply packages/devextreme/js/viz/core/export.js]: fixed

File: packages/devextreme/testing/tests/DevExpress.viz.core/export.tests.js (hunk 2 above)
Comment (alexanderPolosatov) [reply packages/devextreme/testing/tests/DevExpress.viz.core/export.tests.js]: yes) you are right. fixed

**Review by ksercs — APPROVED** [review]

File: packages/devextreme/js/viz/core/export.js [hunk 3]
```diff
@@ -311,7 +342,28 @@ export let combineMarkups = function(widgets, options = { }) {
         return r;
     }, { items: [], rowOffsets: [], rowHeights: [], itemWidth: 0, totalHeight: 0, maxItemLen: 0, totalWidth: 0, backgroundColors: [] });
 
-    const backgroundColor = `data-backgroundcolor="${exportItems.backgroundColors.length === 1 ? exportItems.backgroundColors[0] : '' }" `;
+    const backgroundColor = `${exportItems.backgroundColors.length === 1 ? exportItems.backgroundColors[0] : '' }`;
+    const { totalWidth, totalHeight } = exportItems;
+
+    const rootElement = wrapItemsToElement(totalWidth,
```
Comment (ksercs) [inline packages/devextreme/js/viz/core/export.js:348]: actually I expected moving of a `totalWidth` to the next line as well, but it's not so important, just for your information

**Review by pomahtri — APPROVED** [review]

**Pull request merged**